src/components/ebay-dialog/index.js

```diff
@@ -10,8 +10,10 @@ const template = require('./template.marko');
 
 function init() {
     this.dialogEl = this.getEl('dialog');
+    this.windowEl = this.getEl('window');
```

Comment:
Does listening on `window`'s `transitionend` event wait for all the dialog's children transition to complete? Curios on why you are adding `window` to the wait list.

Reply:
It’s the dialog window, not the actual window. The two elements that have transitions are the dialog itself and the `dialog__window`.